Dockerfile

```
@@ -1,4 +1,7 @@
-FROM gcr.io/distroless/static:nonroot
+FROM alpine
```

Please always use full path and version to include Renovate support
```suggestion
FROM docker.io/library/alpine:3.14
```